tools/shoestring/shoestring/healthagents/rest_https_certificate.py

```diff
@@ -24,25 +26,45 @@ def _openssl_run_sclient_verify(hostname, test_args):
 		'localhost:3001'
 	] + test_args, command_input='Q', show_output=False)
 
+	# verify the certificate is valid
+	for line in lines:
+		line = line.strip()
+		if line.startswith('verify error:'):
+			# normalize error message
+			if '1.1.1' in openssl_executor.version():
+				line = line.replace('self signed', 'self-signed')
+			return False, line
```

Comment:
is it possible to add test for this case?

Reply:
Add a testcase for the normalize message scenario?
There is a test for self-signed -  ``test_validate_fails_when_certificate_is_self_signed``

Reply:
i mean error branch